modules/dag-l1/src/test/scala/org/tessellation/dag/l1/infrastructure/trust/storage/TrustStorageSuite.scala

```diff
@@ -0,0 +1,52 @@
+package org.tessellation.dag.l1.infrastructure.trust.storage
+
+import cats.effect.IO
+
+import org.tessellation.dag.l1.config.types.DBConfig
+import org.tessellation.dag.l1.infrastructure.db.Database
+import org.tessellation.scalacheck.generators.trust.genTrustDbValues
+
+import ciris.Secret
+import eu.timepit.refined.auto._
+import weaver._
+
+object TrustStorageSuite extends SimpleIOSuite {
+
+  val testTrustDbValues = genTrustDbValues.sample.get
+
+  test("Test writing trust values to the database - score.") {
+    val dbConfig = DBConfig("org.sqlite.JDBC", "jdbc:sqlite::memory:", "sa", Secret(""))
```

Comment:
`dbConfig` can be moved out of the test cases to keep it DRY ("don't repeat yourself")

Reply:
It is not. It will be removed.